packages/viewer/src/lib/types/frame.ts

```diff
@@ -356,7 +356,7 @@ export interface FrameCameraWithMatrices extends FrameCameraBase {
 
 export class FramePerspectiveCamera
   extends FrameCameraBase
-  implements FrameCameraWithMatrices, FramePerspectiveCameraLike
+  implements FramePerspectiveCameraLike
```

Comment:
My vote would be to remove these interface definitions as well:

* `FrameCameraLike`
* `FramePerspectiveCameraLike`
* `FrameOrthographicCameraLike`

I think this'd help reduce complexity and confusion.

Reply:
Good idea. Removed!